code/game/objects/buckling.dm

```diff
@@ -1,103 +1,118 @@
 /obj
 	var/can_buckle = 0
+	var/can_buckle_bags = 0
 	var/buckle_movable = 0
 	var/buckle_dir = 0
 	var/buckle_lying = -1 //bed-like behavior, forces mob.lying = buckle_lying if != -1
 	var/buckle_require_restraints = 0 //require people to be handcuffed before being able to buckle. eg: pipes
 	var/mob/living/buckled_mob = null
+	var/obj/structure/closet/body_bag/buckled_bag = null
```

Comment:
Similarly, this variable should then be only an /atom/movable or /obj variable. Though I guess it's possible to unify this with the buckled_mob variable and only have `var/atom/movable/buckled`?